src/screens/Sidebar/Sidebar.res

```diff
@@ -536,6 +545,8 @@ let make = (
     )
   }
 
+  let (openItem, setOpenItem) = React.useState(_ => "")
```

Comment:
declare state in the starting of the component.

Reply:
okay, sure